fast_neural_style/neural_style/utils.py

```diff
@@ -0,0 +1,47 @@
+import torch
+from PIL import Image
+from torch.autograd import Variable
+
+
+def load_image(filename, size=None, scale=None):
+    img = Image.open(filename)
+    if size is not None:
+        img = img.resize((size, size), Image.ANTIALIAS)
+    elif scale is not None:
+        img = img.resize((int(img.size[0] / scale), int(img.size[1] / scale)), Image.ANTIALIAS)
+    return img
+
+
+def save_image(filename, data):
+    img = data.clone().clamp(0, 255).numpy()
+    img = img.transpose(1, 2, 0).astype("uint8")
+    img = Image.fromarray(img)
+    img.save(filename)
+
+
+def gram_matrix(y):
+    (b, ch, h, w) = y.size()
+    features = y.view(b, ch, w * h)
+    features_t = features.transpose(1, 2)
+    gram = features.bmm(features_t) / (ch * h * w)
+    return gram
+
+
+def normalize_batch(batch):
```

Comment:
Can't you use `torchvision.transforms.Normalize` in here? There is the 255 normalization that differs (but I think it could be removed as in my previous comment`, but even if we keep it, it could just be added as a LambdaTransform, right?